pandas/tests/frame/test_sorting.py

```diff
@@ -227,6 +228,30 @@ def test_stable_descending_multicolumn_sort(self):
                                    kind='mergesort')
         assert_frame_equal(sorted_df, expected)
 
+    def test_sort_multi_index(self):
+        # GH 25775
```

Comment:
i am not clear by this example what you are even trying to test here. can you add some comments on the test and make the example a little more fluid (maybe add some comments).

Reply:
updated to the simple example by the other reviewer

Reply:
can you add a comment here on what this is testing, example is nice